dev-libs/wlc/wlc-0.0.2-r1.ebuild

```diff
@@ -48,3 +48,11 @@ src_configure() {
 
 	cmake-utils_src_configure
 }
+
+pkg_postinst() {
+	if use X
```

Comment:
@mgorny 

I didn't know I could do that. I'll do it; thanks for the suggestion.